npc/re/instances/IsleOfBios.txt

```diff
@@ -0,0 +1,769 @@
+//================= Hercules Script =======================================
+//=       _   _                     _
+//=      | | | |                   | |
+//=      | |_| | ___ _ __ ___ _   _| | ___  ___
+//=      |  _  |/ _ \ '__/ __| | | | |/ _ \/ __|
+//=      | | | |  __/ | | (__| |_| | |  __/\__ \
+//=      \_| |_/\___|_|  \___|\__,_|_|\___||___/
+//================= License ===============================================
+//= This file is part of Hercules.
+//= http://herc.ws - http://github.com/HerculesWS/Hercules
+//=
+//= Copyright (C) 2012-2024 Hercules Dev Team
+//= Copyright (C) Euphy
+//= Copyright (C) L0ne_W0lf
+//=
+//= Hercules is free software: you can redistribute it and/or modify
+//= it under the terms of the GNU General Public License as published by
+//= the Free Software Foundation, either version 3 of the License, or
+//= (at your option) any later version.
+//=
+//= This program is distributed in the hope that it will be useful,
+//= but WITHOUT ANY WARRANTY; without even the implied warranty of
+//= MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//= GNU General Public License for more details.
+//=
+//= You should have received a copy of the GNU General Public License
+//= along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//=========================================================================
+//= Isle of Bios
+//================= Description ===========================================
+//= Bios Island is part of the episode 14.3 and is the prerequisite of all 
+//  the other instances found in the Flame Cave.
+//===== Changelogs: =======================================================
+//= 1.0 First version. [Capuche]
+//= 1.1 Convert to Hercules. [Fiction]
+//=========================================================================
+moro_cav,50,64,5	script	Yellow Seed#bios	CLEAR_NPC,{
+		
+	if (BaseLevel < 160) {
+		mes("^ff0000Characters below Level 160 cannot enter this dungeon.^000000");
+		close();
+	}
+	if (questprogress(15006) == 0) {
+		mes("^ff0000You first must meet Wandering Old Man to gain access to this dungeon.^000000");
+		close();
+	}
+	switch( questprogress(15005,PLAYTIME) ) {
+	case 0:
+		if(select("Create Instance", "Cancel") == 2 ) {
```

Comment:
please fix the spacing to align with Hercules style guide - https://github.com/HerculesWS/Hercules/wiki/Coding-Style